yaksh/templates/yaksh/monitor.html

```diff
@@ -115,4 +118,46 @@
 <h4>No Quiz Found</h4>
 {% endif %}
 {% endif %}
+
+<!-- CSV Modal -->
+<div class="modal fade" id="csvModal" role="dialog">
+    <div class="modal-dialog">
+
+    <!-- Modal content-->
+    <div class="modal-content">
+    <div class="modal-header">
+      <button type="button" class="close" data-dismiss="modal">&times;</button>
+      <h3 class="modal-title">Uncheck unwanted columns</h3>
```

Comment:
Maybe you can change this to `Tick\Untick for specific column`.

Reply:
Changed the text.